Model/User.php

```diff
@@ -132,10 +132,12 @@
      * Sets the creation date.
      *
      * @param \DateTime|null $createdAt
+     * @return $this
```

Comment:
>Annotations in phpdocs should be grouped together so that annotations of the same type immediately follow each other, and annotations of a different type are separated by a single blank line.
[[ref]](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/7ef7ea94ccb0d528daa2f428226958e00926b928/README.rst#usage)